### DISCUSSION TITLE
Additional categorisation into regular vs mode water anticyclones, and regular vs thinny cyclones

### DISCUSSION THREAD

**Discussion by dputrasahan** [AntSimi/py-eddy-tracker#211] [General]
Hi,
I'm trying to further subdivide anticyclones into regular anticyclones and mode water anticyclones, and for cyclones to be subdivided into regular cyclones and thinny cyclones, as proposed in [McGillicuddy (2015)](https://journals.ametsoc.org/view/journals/phoc/45/2/jpo-d-14-0221.1.xml). For the anticyclones (where SSH' >0), regular anticyclones can be identified by surface rho'<0 whilst mode water anticyclones can be identified by surface rho'>0. And for cyclones (SSH'<0), regular cyclones have surface rho'>0 and thinny cyclones have surface rho'<0. While the spatial high-pass filtering can be done easily, I'm wondering how to assign these subdivisions onto the existing anticyclones and cyclones files. Perhaps it's adding additional keys to the existing dictionary? Would it then be better to have separate files for regular vs mode water anticyclones, and regular vs thinny cyclones, or should one keep them together as the original? It would also be great to be able to do basic statistics on each of these categories, for example, for a given time where identified eddies are seen in a map, to be able to tell how many mode water anticyclones, regular anticyclones, regular cyclones and thinny cyclones there are. The thing is that the anticyclones can switch between regular and mode water anticyclones throughout the lifetime of the anticyclone. Similar switching for the cyclones is also present. Since they can switch between regular and mode water along an anticyclone track, it would be good to be able to know how long it was a regular anticyclone and how long it was a mode water anticyclone. 
Pardon my ignorance, as python is quite new to me, but I'm trying to see if these criteria can be implemented as it would further expand eddy analysis. Any advise, recommendation or help is much appreciated. Thanks!

**Comment by scallen01** [comment]
Gday,

I can't help with your question, sorry if I got your hopes up.

I was wondering if you had been able to get any of the statistical plots to work through py-eddy-tracker for your own dataset?

I hope this finds you well,

Scott

**Comment by AntSimi** [comment]
You are right you must add a new field you could do with [this method](https://py-eddy-tracker.readthedocs.io/en/latest/_autosummary/py_eddy_tracker.observations.observation.EddiesObservations.html#py_eddy_tracker.observations.observation.EddiesObservations.add_fields) and filled it with your variable. After this operation you will be able to subsample data and do statistics with all method available and describe in this [page](https://py-eddy-tracker.readthedocs.io/en/latest/_autosummary/py_eddy_tracker.observations.observation.EddiesObservations.html#py_eddy_tracker.observations.observation.EddiesObservations) and [this one](https://py-eddy-tracker.readthedocs.io/en/latest/_autosummary/py_eddy_tracker.observations.tracking.TrackEddiesObservations.html#py_eddy_tracker.observations.tracking.TrackEddiesObservations)
